02-when-vector-data-dont-line-up-CRS.Rmd

```diff
@@ -417,11 +420,11 @@ on our data.
 Create a map of the North Eastern United States as follows:
 
 1. Import and plot `Boundary-US-State-NEast.shp`. Adjust line width as necessary.
-2. REPROJECT the layer into UTM zone 18 north.
+2. *Reproject* the layer into UTM zone 18 north.
 3. Layer the Fisher Tower point location `point_HARV` on top of the above plot.
 4. Add a title to your plot.
-5. Add a LEGEND to your plot that shows both the state boundary (line) and the 
-Tower location POINT.  
+5. Add a legend to your plot that shows both the state boundary (line) and the 
```

Comment:
Please be consistent if you change the case 

*legend* should also be italicized.